includes/elements/container.php

```diff
@@ -312,14 +301,8 @@ protected function register_container_layout_controls() {
 					'boxed' => esc_html__( 'Boxed', 'elementor' ),
 					'full' => esc_html__( 'Full Width', 'elementor' ),
 				],
-				'render_type' => 'ui',
-				'selectors' => [
-					'{{WRAPPER}}' => '{{VALUE}}',
-				],
```

Comment:
I have used a prefix_class to avoid additional css. 
The container type isn't responsive, therefore I believe we can use a prefix. 
I don't think there is another way how we could reduce the css for the frontend selector.

The only alternative which I can think off: we can set 'boxed' or 'full width' as default value, and only add additional css for when the other option is selected.

Reply:
No need for this. As I told you, we want to avoid prefix class.
All you have to do here is keep everything as is except remove the `default` value.
The CSS should only include a value of `--width: 100%` on the container by default

Reply:
Understood.